classes/fields/taxonomy.php

```diff
@@ -12,6 +12,7 @@ class PodsField_Taxonomy extends PodsField_Pick {
      * @since 2.0
      */
     public function __construct () {
+	    parent::__construct();
```

Comment:
We'll leave it since we're not doing that anywhere else (like Avatar) since we're not calling the properties via `static::$property` and only using `self::$property` in the various places.